File: 백준/Bronze/2525. 오븐 시계/오븐 시계.java
import java.io.BufferedReader;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String[] inputs = br.readLine().split(" ");
        int Hour = Integer.parseInt(inputs[0]);
        int Minute = Integer.parseInt(inputs[1]);
        int Cook = Integer.parseInt(br.readLine());

        Minute += Cook;

        while (Minute >= 60) {
            Minute -= 60;
            Hour += 1;

            if (Hour == 24) {
                Hour = 0;
            }
        }

        System.out.println(Hour + " " + Minute);
    }
}

File: 백준/Bronze/2525. 오븐 시계/오븐 시계.py
current_time = input().split()
hour = int(current_time[0])
minute = int(current_time[1])
cook_time = int(input())

minute += cook_time

while minute >= 60:
    minute -= 60
    hour += 1
    if hour == 24:
        hour = 0

print(hour, minute)